app/scripts/modules/amazon/package.json

```diff
@@ -2,14 +2,57 @@
   "name": "@spinnaker/amazon",
   "license": "Apache-2.0",
   "version": "0.0.315",
-  "main": "lib/lib.js",
-  "typings": "lib/index.d.ts",
+  "module": "dist/index.js",
+  "typings": "dist/index.d.ts",
   "scripts": {
-    "clean": "../../../../node_modules/.bin/rimraf lib",
-    "lib": "npm run build",
-    "prepublishOnly": "npm run lib",
-    "generate-types": "../../../../node_modules/.bin/tsc --emitDeclarationOnly",
-    "webpack": "node ../../../../node_modules/.bin/webpack",
-    "build": "npm run clean && npm run generate-types && npm run webpack"
+    "clean": "shx rm -rf dist",
+    "prepublishOnly": "npm run build",
+    "build": "npm run clean && spinnaker-scripts build",
+    "dev": "spinnaker-scripts start",
+    "dev:push": "spinnaker-scripts start --push",
+    "lib": "npm run build"
+  },
+  "dependencies": {
+    "@spinnaker/core": "^0.0.593",
+    "@uirouter/angularjs": "1.0.26",
+    "@uirouter/core": "6.0.5",
+    "@uirouter/react": "1.0.3",
+    "@uirouter/react-hybrid": "1.0.2",
+    "angular": "1.6.10",
+    "angular-ui-bootstrap": "2.5.0",
+    "angular2react": "3.0.2",
+    "bootstrap": "3.4.1",
+    "chart.js": "3.0.0-rc.6",
+    "chartjs-adapter-luxon": "1.0.0-beta.2",
+    "classnames": "2.2.6",
+    "enzyme": "3.10.0",
+    "formik": "1.5.1",
+    "lodash": "4.17.21",
+    "ngimport": "0.6.1",
+    "react": "16.14.0",
+    "react-bootstrap": "0.32.1",
+    "react-ga": "2.4.1",
+    "react-select": "1.2.1",
+    "react-sortable-hoc": "0.6.8",
+    "react-virtualized-select": "3.1.3",
+    "react2angular": "3.2.1",
+    "rxjs": "6.6.7"
```

Comment:
What do you think of making these deps be transitive from `@spinnaker/core`?  Two thoughts: 
- you shouldn't directly reference a package (like import react from React) if it's not in the package.json
- this will be a lot of duplicated dependencies in all the packages.  this can be managed via tooling.

Reply:
Not sure I follow. The packages listed here are directly referenced in `@spinnaker/amazon`. (expect `react-ga` which I think was removed recently).